pkg/logs/config/integration_config.go

```diff
@@ -70,6 +70,17 @@ type IntegrationConfig struct {
 	Logs []IntegrationConfigLogSource
 }
 
+// GetWildcardOpenFilesLimit returns the maximum number of files that can be opened at the same time
+func GetWildcardOpenFilesLimit() int {
+	return getWildcardOpenFilesLimit(LogsAgent)
+}
+
+// getWildcardOpenFilesLimit returns the value specified in the configuration file,
+// if it does not exist, it returns 0 by default
+func getWildcardOpenFilesLimit(config *viper.Viper) int {
+	return config.GetInt("wildcard_open_files_limit")
+}
```

Comment:
is it worth adding those lines of code vs a simple `config.GetInt("wildcard_open_files_limit")`? it's only used once

Reply:
I think we could remove this part but I would keep `GetWildcardOpenFilesLimit` anyways because I don't want any other package to know about `viper`.